nautobot/core/tests/test_views.py

```diff
@@ -338,6 +338,11 @@ def query_and_parse_metrics(self):
         page_content = response.content.decode(response.charset)
         return text_string_to_metric_families(page_content)
 
+    @override_settings(METRICS_AUTHENTICATED=True)
+    def query_and_parse_metrics_authenticated(self):
+        response = self.client.get(reverse("metrics"))
+        self.assertHttpStatus(response, 403, msg="/metrics should return a 403 HTTP status code.")
```

Comment:
Why is a 403 expected here? Since this test is a subclass of `nautobot.core.testing.TestCase`, `self.client` should be logged in already. What am I missing?

Reply:
You aren't missing anything. I had mainly copied the other test and assumed it worked since the tests passed. But it didn't.

I redid the tests. Basically, now in the tests, `METRICS_AUTHENTICATED=True` so we can validate that not being logged in gives a 403 and it tests that the unauthenticated view returns a 200.